pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -517,8 +515,8 @@ func (b *volumeBinder) bindAPIUpdate(ctx context.Context, pod *v1.Pod, bindings
 	// Update claims objects to trigger volume provisioning. Let the PV controller take care of the rest
 	// PV controller is expected to signal back by removing related annotations if actual provisioning fails
 	for i, claim = range claimsToProvision {
-		klog.V(5).InfoS("Updating claims objects to trigger volume provisioning", "pod", klog.KObj(pod), "PVC", klog.KObj(claim))
```

Comment:
Why removing `pod:klog.KObj(pod)` pair? Is it already set in the logger some place else?